TAG

```diff
@@ -0,0 +1 @@
+v0.24.0-dev.0
```

Comment:
Look into github `post-checkout` hook to populate this file. Hooks look pretty cool and easy.